docs/source/index.md

```diff
@@ -22,7 +22,7 @@ General index <genindex>
 Module index <modindex>
 Search <search>
 
-[Source code](https://github.com/DIDONEproject/musif/)
```

Comment:
Sphynx `toctree` also uses [reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#external-links), not Markdown.